src/main/java/jenkins/scm/impl/subversion/SVNRepositoryView.java

```diff
@@ -84,7 +89,10 @@ public int getReadTimeout(SVNRepository repository) {
         repository.setTunnelProvider(SVNWCUtil.createDefaultOptions(true));
         repository.setAuthenticationManager(sam);
         try {
-            uuid = repository.getRepositoryUUID(false);
+            uuid = repository.getRepositoryUUID(true);
+            if (uuid == null) { // TODO is this even possible? Javadoc is unclear.
```

Comment:
Noticed when running `SCMBinderTest.exactRevisionSubversion` that `uuid` was null, which certainly seemed wrong in this context—that would mean every source would be sharing a single cache! Not sure if this problem is limited to local file connections (as opposed to HTTPS), but seems best to require a defined UUID.